securedrop_client/gui/widgets.py

```diff
@@ -661,6 +683,66 @@ def clear_conversation(self):
                 child.widget().deleteLater()
 
 
+class EmptyConversationView(QWidget):
+
+    CSS = '''
+    #content {
+        font-family: Orbitron;
+        font-size: 35px;
+        color: #a5b3e9;
+        qproperty-alignment: AlignLeft;
+    }
+    '''
+
+    def __init__(self):
+        super().__init__()
+
+        self.setObjectName('view')
+
+        # Set styles
+        self.setStyleSheet(self.CSS)
+        font = QFont("Orbitron", 35, 1)
+        self.setFont(font)
+
+        # Set layout
+        layout = QHBoxLayout(self)
+        self.setLayout(layout)
+
+        # Remove margins
+        layout.setContentsMargins(0, 100, 0, 0)
+        layout.setSpacing(0)
+
+        self.content = QLabel(self)
+        self.content.setObjectName('content')
+        self.content.setWordWrap(True)
+
+        # Set font of content
+        self.content.setFont(font)
+
+        content_layout = QVBoxLayout()
+        self.content.setLayout(content_layout)
+        content_layout.addStretch(1)
+        content_layout.addWidget(self.content, 8)
```

Comment:
this PR works as advertised and the diff looks good, but do you also get this log line `QLayout: Cannot add parent widget QLabel/content to its child layout QVBoxLayout/` ?

Reply:
good catch, this is a bug